### DISCUSSION TITLE
欢迎来到 OSSDevGov2021 课程的讨论

### DISCUSSION THREAD

**Discussion by will-ww** [X-lab2017/OSSDevGov2021#1] [任意分享]
## 👋 Welcome!
  这里是《开源软件开发与社区治理》课程的社区讨论板块. 在这里，我们希望大家能够:
  * 提出你关于课程上的任何问题；
  * 分享你的 idea；
  * 和其它同学来一起交流与互动；
  * 以一个开放的态度欢迎其它成员；
  * 记住：这里是我们共同的社区，大家一起来建设好！ 💪

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by frank-zsy** [comment]
从软件工程演变历史来看开源的必然性是非常有趣和有启发性的，也正是纵向一体化使软件工程的外延在快速扩大，从而延展到其他领域，所以对开源的理解也需要进一步扩展的更大的范围。但就开发侧而言，深入理解软件工程的演化史对理解开源有很重要的意义。
